spanner/row_test.go

```diff
@@ -2067,7 +2068,7 @@ func TestSelectAll(t *testing.T) {
 					mockIterator.On("Next").Once().Return(&Row{
 						[]*sppb.StructType_Field{
 							{Name: "Col1", Type: intType()},
-							{Name: "Col2", Type: floatType()},
+							{Name: "COL2", Type: floatType()},
```

Comment:
Shouldn't this be kept as `Col2` to verify that the string matching is case-insensitive?